solutions/swb-reference/SWBv2.postman_collection.json

```diff
@@ -36,13 +36,8 @@
             },
             "url": {
               "raw": "{{API_URL}}/costCenters/",
-              "host": [
-                "{{API_URL}}"
-              ],
-              "path": [
-                "costCenters",
-                ""
-              ]
+              "host": ["{{API_URL}}"],
```

Comment:
What was the change in this file except formatting?

Reply:
No other changes